formats/tokens.js

```diff
@@ -0,0 +1,19 @@
+const {
+  colorFactory,
+  tokensToColors,
+} = require('./utils/color-factory/color-factory');
+
+function tokenify(variants, scheme) {
```

Comment:
This is no longer a formatting function, but now a higher order function which returns a formatter function. I did this so we can control both which variants and scheme (dark/light) are passed to the factory without having to create n number of formats for 2 modes * n variants.